src/main/java/de/hpi/bpt/argos/eventProcessing/creation/EventFactoryImpl.java

```diff
@@ -0,0 +1,125 @@
+package de.hpi.bpt.argos.eventProcessing.creation;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import de.hpi.bpt.argos.common.EventPlatformFeedback;
+import de.hpi.bpt.argos.common.EventPlatformFeedbackImpl;
+import de.hpi.bpt.argos.common.EventProcessingPlatformUpdater;
+import de.hpi.bpt.argos.common.RestRequest;
+import de.hpi.bpt.argos.common.RestRequestFactoryImpl;
+import de.hpi.bpt.argos.storage.dataModel.event.type.EventType;
+import de.hpi.bpt.argos.util.Pair;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.TimeZone;
+
+/**
+ * {@inheritDoc}
+ * This is the implementation.
+ */
+public final class EventFactoryImpl implements EventFactory {
+	private static final Gson serializer = new Gson();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "serializer" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1068)